nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/src/main/java/org/apache/nifi/processors/mongodb/AbstractMongoProcessor.java

```diff
@@ -268,7 +268,7 @@ protected WriteConcern getWriteConcern(final ProcessContext context) {
     protected void writeBatch(String payload, FlowFile parent, ProcessContext context, ProcessSession session,
             Map<String, String> extraAttributes, Relationship rel) throws UnsupportedEncodingException {
         String charset = parent != null ? context.getProperty(CHARSET).evaluateAttributeExpressions(parent).getValue()
```

Comment:
@MikeThomsen I may be wrong but i think this is supposed to be getting rid of the check for `parent != null`, isn't it? As-is, it's saying if parent != null then evaluate with parent flowfile, otherwise still evaluate with parent flowfile... should just be `String charset = context.getProperty(charset).evaluateAttributeExpressions(parent).getValue()`, no?

Reply:
Yeah, that was a typo on my parent. It should have been `evaluateAttributeExpressions()`